userdocs/profiles.dev/docs/intro.md

```diff
@@ -4,8 +4,128 @@ sidebar_position: 1
 
 # Introduction
 
-What is a profile?
+Welcome to Profiles: _The_ GitOps Native Package Manager :sparkles:
 
-What problem is the profiles mechanism trying to solve?
+:::danger _(highway to the)_ Danger Zone
+Profiles is still highly experimental and likely to experience sudden and significant
+API change as we continue to develop based on incoming feedback.
 
-etc
+Please always refer to the docs and release notes, and alert us to any issues
+or inconsistencies.
+
+Thank you!
+:::
+
+## What are Profiles?
+
+In short, a profile is a Kubernetes package. Much as you would expect from another package
+manager (`brew`, `apt`, etc), the Profiles mechanism allows Kubernetes operators
+to manage their clusters in the same way they manage their host systems. Available
+packages are listed somewhere searchable; they are versioned; they are tested and verified;
+they are installable by way of a single simple command; they are updatable and upgradable.
+Moreover they are reliable and uniformly manageable through a simple intuitive cli tool.
+
+Sticking with the `apt` analogy, let's look at one of the most basic and ubiquitous
+packages: `coreutils`. By installing `coreutils`, users understand that they are
+installing a bulk set of the most commonly used and useful tools on their unix system.
+They don't care much what the full roster is, just that the thing is now more operable
+than it was before; that the various pieces work; that the whole thing can be updated
+or removed just as simply.
+
+Profiles provides the same service for cluster operators. On spinning up a new cluster or
+fleet of clusters, operators can then apply a standard set of profiles to get things
+to a uniform, operable standard. They could choose to install, say, a standard Observability
+package, maybe a Logging set too. If they have some in-house collections, they are able to create
+their own custom profiles to bundle these up, and apply them across their infrastructure.
+They even have the option of creating a single "install all the things" profile to bring
+their most commonly used collections under one reliable and reusable package, giving them just one
+thing to do to bootstrap a new cluster to their org specifications.
+
+As part of the WeaveGitOps ecosystem, Profiles is designed from its core to follow the best practices
+of GitOps, that is; managing infrastructure as code. Profiles therefore uses [Flux](https://fluxcd.io/)
+to fit into users' existing cluster management practices.
+
+:::tip Heard the word "profiles" too much?
+No worries, here's the breakdown:
+
+- A profile: an individual package of Kubernetes components. Lives in a git repo "upstream"
+  of users who have installed it on their cluster
+- `N` profiles: more than one profile
```

Comment:
This is a weird sentence on its own. N profiles where? In what format? And what does that mean? Is this just to say, N profiles means more than one profile? Maybe replace `:` with `means`? :)

Reply:
lol i thought everyone got that '`N` blah' means 'more than one of blah' 😂

Reply:
Fair enough. 😂